libs/httpext/src/lib/http-ext.spec.ts

```diff
@@ -58,15 +58,18 @@ describe('HttpExt', () => {
       })
     );
 
-    response$.subscribe(resp => {
-      expect(resp).toEqual({
-        data: { answer: 42 },
-        status: 200,
-        statusText: 'OK',
-        headers: {}
-      });
-      done();
-    });
+    const responseObserver = jest.fn();
+
+    response$.subscribe(responseObserver);
+
+    expect(responseObserver).toHaveBeenCalledTimes(1);
+    expect(responseObserver).toHaveBeenCalledWith(
+      expect.objectContaining({
+        data: {
+          answer: 42
+        }
+      })
+    );
```

Comment:
Using a mock observer we can count the calls and make the test synchronous.
As we're not expecting more than one call we could also use async / await and just do something like `response = response$.toPromise()`. The drawbacks are that we can't count the emitted values and as specs are doc, it's not the best way to understand how things work.